ckanext/dgu/theme/templates/issues/all_issues.html

```diff
@@ -1,54 +1,118 @@
+      <span class="input-group-btn">
+        <button class="btn btn-primary" type="submit"><i class="icon-search"></i></button>
+      </span>
+   </div>
+
+    <span>
+      <input type="hidden" name="page" value="1">
+      <input type="hidden" name="per_page" value="{{pagination.per_page}}">
+      <input type="hidden" name="status" value="{{status}}">
+    </span>
+
+    <div class="form-select control-group control-order-by pull-right" style="margin-top:10px;">
+      <label for="field-order-by">Order by</label>
+      <select id="field-order-by" name="sort">
+            <option value="least_commented" {% if sort == "least_commented" %}selected="selected"{% endif%}>Least Commented</option>
+            <option value="least_recently_updated" {% if sort == "least_recently_updated" %}selected="selected"{% endif%}>Least Recently Updated</option>
+            <option value="most_commented" {% if sort == "most_commented" %}selected="selected"{% endif%}>Most Commented</option>
+            <option value="newest" {% if sort == "newest" %}selected="selected"{% endif%}>Newest</option>
+            <option value="oldest" {% if sort == "oldest" %}selected="selected"{% endif%}>Oldest</option>
+            <option value="recently_updated" {% if sort == "recently_updated" %}selected="selected"{% endif%}>Recently Update</option>
+        </select>
+        <button class="btn js-hide" type="submit">Go</button>
+      </div>
+    </form>
+    <div class="clearfix"></div>
+
+      <h2>
+        {{ ungettext('{number} issue found', '{number} issues found', pagination.total_count) .format(number=pagination.total_count) }}
+      </h2>
+    </div>
+    <div id="issue-page">
+      {% if issues %}
+        <div>
+          <table class="table">
+            <thead>
+              <tr>
+                <th></th>
+                <th>Title</th>
+                <th>Dataset</th>
+                <th>Reported</th>
+                <th>Status</th>
+                <th>Comments</th>
+              </tr>
+            </thead>
+            <tbody>
+            {% for issue in issues %}
+             {% set issue_url = h.url_for('issues_show',dataset_id=issue.dataset.name,issue_number=issue.number) %}
+              <tr class="{{'success' if issue.resolved else 'error'}}">
+                <td>
+                  <a href="{{ issue_url }}">
+                    <span class="list-group-item-number">#{{issue.id}}</span>
+                  </a>
+                </td>
+                <td>
+                  <a href="{{ issue_url }}">
+                    {{issue.title}}
+                  </a>
+                  {% if issue.resource %}
+                    <br/><br/>
+                    Resource: {{issue.resource.name or issue.resource.description}}
+                  {% endif %}
+                  {% if issue.visibility == 'hidden' %}
+                    <br/><br/>
+                    <div class="issue-comment-label">
+                      Reported as spam/abuse
```

Comment:
Not quite. issue.visibility refers to whether it has been decided (through moderation or repeated reports, etc) that it is spam/abuse. The logic is all in macro issue_item() - can you just use that perhaps?